src/Core/Components/Search/FluentSearch.razor.cs

```diff
@@ -62,6 +62,13 @@ public partial class FluentSearch : FluentInputBase<string?>
     [Parameter]
     public FluentInputAppearance Appearance { get; set; } = FluentInputAppearance.Outline;
 
+    /// <summary>
+    /// Specifies whether a form or an input field should have autocomplete "on" or "off" or another value.
```

Comment:
Can you start the comment using `Gets or sets...`? To be coherent with the other properties?